src/queue.jl

```diff
@@ -4,6 +4,27 @@
     Queue{T}([blksize::Integer=1024])
 
 Create a `Queue` object containing elements of type `T`.
+
+# Examples
+```jldoctest
+julia> q = Queue{Int}()
+Queue{Int64}(Deque [Int64[]])
+
+julia> push!(q, 1)
+Queue{Int64}(Deque [[1]])
+
+julia> x = top(q)
```

Comment:
`top` is deprecated

```suggestion
julia> x = first(q)
```

Reply:
Sorry, should have caught that. I've fixed it in the Queue and Stack docstrings.